### DISCUSSION TITLE
PC crash after making the game fullscreen

### DISCUSSION THREAD

**Discussion by jojopipe** [ppy/osu#21836] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

when i hit f11 to make the game fullscreen since it was in windowed mode, first the game crashed, then my whole PC just froze and eventually crashed afer ~10 secs. Got a windows bluescreen with "VIDEO_SCHEDULE_INTERNAL_ERROR".
this has happened to me before, but then it didn't crash my PC.

Windows 10
intel i7 8700K
nvidia geforce 1060 6gb

### Version

2022.1227.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10310546/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/10310547/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/10310548/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10310549/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10310550/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10310551/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10310552/updater.log)
[database.log](https://github.com/ppy/osu/files/10310553/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10310554/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10310555/network.log)

**Comment by peppy** [comment, marked as answer]
Driver issue. Please try otehr drivers or wait for opengl to be replaced in an upcoming release, which will likely fix this in a roundabout way.
